### ISSUE TITLE
Image resize for training

### ISSUE THREAD

**Issue by QuickLearner171998** [Wovchena/text-detection-fots.pytorch#18]
Hi, I did not find an option to resize images for training.

Also while inferencing if I resize the images to 384 by changing these lines
https://github.com/Wovchena/text-detection-fots.pytorch/blob/04b969c87491630dce38cdb48932aa33115798c6/test.py#L20

nothing is predicted using the pretrained model.

**Comment by Wovchena** [comment]
Training images are upscaled here: https://github.com/Wovchena/text-detection-fots.pytorch/blob/04b969c87491630dce38cdb48932aa33115798c6/datasets.py#L32

I think the model doesn't work for size of 384 because it was trained for a larger scale. You may also try visualizing raw net outputs. May be it will give you some understanding why it happens.

**Comment by QuickLearner171998** [comment]
Hey I tried training on images with size 384x384 , train loss curve was also fine.
But if I resize test images(as mentioned above) the results are incorrect and when I keep the code scales as it is I get decent results. Any idea why is this happening?

I notice here:
https://github.com/Wovchena/text-detection-fots.pytorch/blob/04b969c87491630dce38cdb48932aa33115798c6/datasets.py#L25

IN_SIDE = 640 does that mean the input size in training is 640?

**Comment by QuickLearner171998** [comment]
> Training images are upscaled here:
> 
> https://github.com/Wovchena/text-detection-fots.pytorch/blob/04b969c87491630dce38cdb48932aa33115798c6/datasets.py#L32
> 
> I think the model doesn't work for size of 384 because it was trained for a larger scale. You may also try visualizing raw net outputs. May be it will give you some understanding why it happens.


Hi I am trying to visualize the gt masks...I changed this line:
https://github.com/Wovchena/text-detection-fots.pytorch/blob/04b969c87491630dce38cdb48932aa33115798c6/train.py#L211

to if 1
Only black images are shown and now there is div by zero error while calculating dist.
Also, can you explain how can I visualize the masks in between training?
